module/dkg/client.go

```diff
@@ -33,13 +33,17 @@ type Client struct {
 func NewClient(
 	log zerolog.Logger,
 	flowClient module.SDKClientWrapper,
+	flowClientANID flow.Identifier,
 	signer sdkcrypto.Signer,
 	dkgContractAddress,
 	accountAddress string,
 	accountKeyIndex uint,
 ) *Client {
 
-	log = log.With().Str("component", "dkg_contract_client").Logger()
+	log = log.With().
+		Str("component", "dkg_contract_client").
+		Str("flow-client-an-id", flowClientANID.String()).
```

Comment:
```suggestion
		Str("flow_client_an_id", flowClientANID.String()).
```
Nitpick: use same casing as above

Reply:
https://github.com/onflow/flow-go/pull/2263/commits/3b67687eae3a7501cf779878248295092a0dc604